wdio.conf.js

```diff
@@ -7,94 +7,111 @@ const GITHUB_REF = process.env.GITHUB_REF || '';
 const BUILD = GITHUB_SHA || GITHUB_REF ? `${GITHUB_REF} ${GITHUB_SHA}` : 'manual build';
 
 const commonCapabilities = {
-  project: PROJECT,
-  build: BUILD,
-  name: 'browser test',
-  'browserstack.local': true,
-  'browserstack.debug': false,
-  'browserstack.console': 'errors',
+  projectName: PROJECT,
+  buildName: BUILD,
+  sessionName: 'browser test',
   resolution: '1024x768',
 };
 
 const capabilities = [
   // IE
   {
-    ...commonCapabilities,
+    'bstack:options': {
+      os: 'Windows',
+      osVersion: '10',
+      ...commonCapabilities,
+    },
     browserName: 'IE',
-    browser_version: '11.0',
-    os: 'Windows',
-    os_version: '10',
+    browserVersion: '11.0',
   },
   {
-    ...commonCapabilities,
+    'bstack:options': {
+      ...commonCapabilities,
+      os: 'Windows',
+      osVersion: '7',
+    },
     browserName: 'IE',
-    browser_version: '11.0',
-    os: 'Windows',
-    os_version: '7',
+    browserVersion: '11.0',
   },
 
   // Chrome
   {
-    ...commonCapabilities,
+    'bstack:options': {
```

Comment:
Latest `main` have another entry here, just adding this comment so we don't miss updating that when merging/rebasing. I think that the merge/rebase will apply cleanly, but there will be one entry not updated to the new format ☺️